packages/ra-ui-materialui/src/layout/Sidebar.js

```diff
@@ -11,7 +11,7 @@ export const CLOSED_DRAWER_WIDTH = 55;
 const useStyles = makeStyles(
     theme => ({
         drawerPaper: {
-            position: 'relative',
+            position: 'sticky',
```

Comment:
```suggestion
            position: 'fixed',
```
please take a look at this doc for explanation. https://caniuse.com/#feat=css-fixed

Reply:
But changing 'sticky' for 'fixed' by itself doesn't do anything. I already tried it.

Reply:
I looked at your PR. and try to add

position: 'fixed',
top: '4em',
zIndex: 100,

but it breaks the layout.

Reply:
It works in the demo because you have a surrounding div, but not when changing the same in Sidebars's Drawer component

Reply:
Any other suggestions?